wagtail_review/models.py

```diff
@@ -152,12 +154,14 @@ def share(self):
 
     def can_review(self):
         """
-        Returns True if the workflow is in a ReviewTask and the reviewer is one of the reviewers.
+        Returns True if the workflow is in a ReviewTask or GroupReviewTask and the reviewer is one of the reviewers.
         """
-        if isinstance(self.page.current_workflow_task, ReviewTask):
-            reviewers = self.page.current_workflow_task.reviewers
-            if reviewers.filter(pk=self.reviewer.pk).exists():
+        try:
+            actions = {action[0] for action in self.page.current_workflow_task.get_actions(self.page, user=None, reviewer=self.reviewer)}
```

Comment:
If `current_workflow_task` is defined for this page, but is something other than ReviewTask / GroupReviewTask, then its `get_actions` method will not accept a `reviewer` argument, and this will fail with a TypeError (which doesn't get caught). I think it's best to keep the type check in place here.

Reply:
I've added an except for the TypeError, would this work? I agree the type check works currently, but ideally I'd like to make review functionality usable by custom tasks via the mixin factored out in the next PR, https://github.com/wagtail/wagtail-review/pull/27/. Otherwise I could check for inheriting from that mixin, I suppose?

Reply:
Sounds good - yep, I think that change is worth making in #27. I'm a bit wary about catching TypeError, as that could mask legitimate errors too - I guess the same is true to a lesser extent for AttributeError too, so it would be good to change that to an `if self.page.current_workflow_task:` (assuming that's the case it's meant to be catching...)